mermaid.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: "Mermaid diagrams"
+description: "Display diagrams using [Mermaid](https://mermaid.js.org/)"
```

Comment:
Don't need two links that reference mermaid.js. Can probably remove the link on the description